packaging/ubuntu/postinst

```diff
@@ -18,8 +18,8 @@
 set -e
 
 directories="
-    /var/lib/sawtooth-poet2
-    /var/log/sawtooth-poet2
+    /var/lib/sawtooth-poet
+    /var/log/sawtooth-poet
```

Comment:
[comment not related to this PR] This location can be used instead of "log/xyz" in main.rs of poet2